source/loader/layers/validation/backtrace_libbacktrace.cpp

```diff
@@ -42,14 +42,14 @@ int backtrace_cb(void *data, uintptr_t pc, const char *filename, int lineno,
     } else if (function != NULL) {
         backtraceLine << "(" << function << ") ";
     } else {
-        backtraceLine << "(????????) ";
+        backtraceLine << "(???????\?) ";
```

Comment:
Add a comment explaining that we escape the `?` sign to avoid a trigraph compiler warning.